geotrek/tourism/models.py

```diff
@@ -387,7 +388,7 @@ def __str__(self):
 
 
 class TouristicEventOrganizer(TimeStampedModelMixin):
-    label = models.CharField(verbose_name=_("Label"), max_length=256)
+    label = models.CharField(verbose_name=_("OrganizerLabel"), max_length=256)
```

Comment:
if you change a tanslation, you should update traslation files and set french translation at least.

Byt maybe it's a mistake

Reply:
Ok.
Nop not a mistake. The `Label` has already a translation which was not appropriate for this context